opcua_tools/nodeset_parser.py

```diff
@@ -26,6 +27,8 @@
 from opcua_tools.ua_data_types import UANodeId
 from opcua_tools.value_parser import parse_nodeid, parse_value
 
+from .type_hints import ModelLine, ModelsLine, NameSpaceURIsLine, UANodeSetLine

```

Comment:
I prefer to stick to one import convention - absolute imports instead of the relative ones.

Reply:
ok